src/openms/include/OpenMS/ANALYSIS/OPENSWATH/SwathMapMassCorrection.h

```diff
@@ -94,6 +94,56 @@ namespace OpenMS
     std::vector<OpenSwath::SwathMap> findSwathMapsPasef(const OpenMS::MRMFeatureFinderScoring::MRMTransitionGroupType& transition_group,
                                                          const std::vector< OpenSwath::SwathMap > & swath_maps);
 
+    /**
+      @brief Estimate an extraction window from absolute residuals.
+
+      Treats the input @p residuals as absolute errors (e.g., |Δppm| for m/z).
```

Comment:
not sure if all UTF-8 characters are correctly rendered / processed by doxygen...